PredicateKit/CoreData/NSManagedObjectContextExtensions.swift

```diff
@@ -370,6 +370,28 @@ public struct FetchRequest<Entity: NSManagedObject> {
     let request: NSFetchRequest<Entity> = requestBuilder.makeRequest(from: self)
     return try context.count(for: request)
   }
+  
+   /// An NSFetchedResultsController initialized with the fetch request.
```

Comment:
```suggestion
   /// Returns an NSFetchedResultsController initialized with the fetch request.
```

Reply:
In order to get all unit tests running, I had to:
1. Add a test target to Package.swift
2. In NSManagedObjectContextExtensionsTests, use `Bundle.module` instead of `Bundle(for: NSManagedObjectContextExtensionsTests.self)]` to retrieve DataModel
3. In DataModel.xcdatamodeld, change the module on each Entity to `PredicateKitTests` instead of `Current Product Module`

Do you want me to back those changes out before committing?